HandHistories.Parser/Parsers/FastParser/GGPoker/GGPokerFastParserImpl.cs

```diff
@@ -0,0 +1,897 @@
+﻿using HandHistories.Parser.Parsers.Base;
+using HandHistories.Parser.Parsers.FastParser.Base;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using HandHistories.Objects;
+using HandHistories.Objects.Actions;
+using HandHistories.Objects.Cards;
+using HandHistories.Objects.GameDescription;
+using HandHistories.Objects.Hand;
+using HandHistories.Objects.Players;
+using HandHistories.Parser.Parsers.Exceptions;
+using HandHistories.Parser.Utils.Extensions;
+using HandHistories.Parser.Utils.FastParsing;
+using HandHistories.Parser.Utils.Uncalled;
+using System.Globalization;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Net.NetworkInformation;
+using System.Diagnostics;
+using System.ComponentModel;
+using System.Net;
+
+namespace HandHistories.Parser.Parsers.FastParser.GGPoker
+{
+
+    public partial class GGPokerFastParserImpl : HandHistoryParserFastImpl, IThreeStateParser
+    {
+        static readonly TimeZoneInfo PokerStarsTimeZone = TimeZoneInfo.FindSystemTimeZoneById("Eastern Standard Time");
+
+        public override bool RequiresAdjustedRaiseSizes => true;
+
+        public override bool SupportRunItTwice => true;
+
+        public override bool RequiresActionSorting => true;
+
+        public override SiteName SiteName => SiteName.GGPoker;
+
+        private readonly NumberFormatInfo _numberFormatInfo;
+
+        private readonly Regex _handSplitRegex;
+
+        private readonly Regex _handIdRegex;
+
+        private readonly String _summarySeperator;
```

Comment:
Consider making _summarySeperator a constant or inlining it.

Reply:
Done.